packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputElement.tsx

```diff
@@ -187,11 +187,16 @@ export const TransactionReviewOutputElement = forwardRef<
                                             <FormattedCryptoAmount
                                                 disableHiddenPlaceholder
                                                 value={line.value}
-                                                symbol={cryptoSymbol}
+                                                symbol={
+                                                    line.id !== 'fee' && token
+                                                        ? token.symbol
+                                                        : cryptoSymbol
+                                                }
                                             />
                                         )}
                                     </OutputValueWrapper>
-                                    {fiatVisible && (
+                                    {/* temporary solution until fiat value for ERC20 tokens will be fixed  */}
+                                    {fiatVisible && !(line.id !== 'fee' && token) && (
```

Comment:
Hide it instead of showing `0.00` (or add sign `>0.00`. Probably for another issue)
![Screenshot 2023-10-31 at 10 31 18](https://github.com/trezor/trezor-suite/assets/33235762/d04be2e5-3fe3-427d-a12b-24d722120eb9)

Reply:
let's not touch this, it's not in the lates flow anymore